vmtest-haskell

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+KRUN_BINARY=".build/rv-k/k-distribution/target/release/k/bin/krun"
+KOMPILED_DEF=".build/haskell/"
+HASKELL_BACKEND_CMD=".build/kore/bin/kore-exec"
+
+cSCHEDULE=DEFAULT
+cMODE=VMTESTS
+
+PGM=$1
+
+$KRUN_BINARY --directory "$KOMPILED_DEF" -cSCHEDULE="$cSCHEDULE" -pSCHEDULE='printf %s' -cMODE="$cMODE" -pMODE='printf %s' "$PGM" --haskell-backend-command "$HASKELL_BACKEND_CMD"
```

Comment:
setting the parser to printf %s means that you are expected to pass kast to the -c flag, but you're not here.

Reply:
Understood. I saw that used in KEVM, wasn't sure what it was for but for some reason I thought it was needed. Removed now! 

By the way, I used to get the same error regardless of what test I was trying to run, but now it is gone, so it was probably caused by those bad command line settings.